CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 
 - Lazily CharacterSet only once in SentryBaggageSerialization (#5871)
 - Structured Logging: Log `SentrySDK.logger` calls to `SentrySDKLog` (#5991)
+- Build type metadata now makes difference between `enterprise` and `adhoc` (#6044)
```

Comment:
`m`: We can be a bit more specific so the user knows exactly where this surfaces.
```suggestion
- The build type in the app context now differentiates between `enterprise` and `adhoc` (#6044)
```